subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/SimpleGlobalFilesBuildSettingsDescriptor.java

```diff
@@ -35,6 +35,7 @@ public void generate(InitSettings settings) {
                     + "Detailed information about configuring a multi-project build in Gradle can be found\n"
                     + "in the user manual at " + documentationRegistry.getDocumentationFor("multi_project_builds"))
             .propertyAssignment(null, "rootProject.name", settings.getProjectName())
+            .methodInvocation(null, "include", settings.getSubprojectName())
```

Comment:
I think we should include a comment here to point new users to look in the subdirectory for the "real" part of the project.